.github/actions/bench/action.yml

```diff
@@ -79,6 +79,10 @@ runs:
               --cflags="${{ inputs.cflags }}" --arch-flags="${{ inputs.archflags }}" \
               $([[ ${{ inputs.opt }} == "false" ]] && echo "--no-opt")  \
               -v --output=output.json ${{ inputs.bench_extra_args }}
+    - name: Check namespace
```

Comment:
That doesn't seem to belong in the bench tests.

Reply:
Do we test all different variants of the asm (esp. Keccak) in the functional tests? 
I was afraid that we may not, so I added it here. It's basically free.

Reply:
Okay.